web/models/abilities.ex

```diff
@@ -0,0 +1,9 @@
+defmodule Canary.Abilities do
+  alias CodeCorps.{User}
+
+  defimpl Canada.Can, for: User do
+    def can?(current_user, action, user = %User{}) when action in [:update] do
+      current_user.id == user.id
+    end
```

Comment:
I'd like to point out that calling this method the elixir way would look like

```Elixir
current_user |> can? :update, user
```

Which is just awesome!